tests/api/test_views.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
+
+import pytest
+from django.urls import reverse
+from django_webtest import DjangoTestApp
+
+from vitrina.api.factories import APIKeyFactory
+from vitrina.catalogs.factories import CatalogFactory
+from vitrina.classifiers.factories import CategoryFactory, LicenceFactory
+
+
+@pytest.mark.django_db
+def test_retrieve_catalog_list_without_api_key(app: DjangoTestApp):
+    res = app.get(reverse("catalog-list"), expect_errors=True)
```

Comment:
Can all api related url names be siffixed with `-api`? In order to avoid clashes with non-api views? In this case it could be:

```suggestion
    res = app.get(reverse("catalog-list-api"), expect_errors=True)
```

Reply:
Url names in router are formed like this: {basename}-{action}, so I could only add an api prefix to basename.